src/Reducers/requestAssetsReducers.js

```diff
@@ -62,7 +62,27 @@ const requestAssetsReducers = (state = initialState.requestAssets, action) => {
     case 'HANDLE_SELECTED_EMPLOYEES': {
       return { ...state, selectedEmployees: action.payload };
     }
-
+    case 'HANDLE_MODAL_OPEN': {
+      return { ...state, openModal: !state.openModal };
+    }
+    case 'HANDLE_MODAL_CLOSE': {
+      const searchResultCopy = state.selectedSearchResultCopy.reduce(
```

Comment:
This seems like it would be better suited for a filter to understand the functionality. I'm thinking something like: `state.selectedSearchResultCopy.filter(value => !value.chec)`

Reply:
i  like using reduce. But i guess it would be better suited for to use filter here .I will update it.